sdks/router-sdk/src/entities/mixedRoute/trade.ts

```diff
@@ -48,7 +51,7 @@ export function tradeComparator<TInput extends Currency, TOutput extends Currenc
  * Represents a trade executed against a set of routes where some percentage of the input is
  * split across each route.
  *
- * Each route has its own set of pools. Pools can not be re-used across routes.
+ * Each route has its own set of pools. V3Pools can not be re-used across routes.
```

Comment:
& V4Pools I assume?

Reply:
hahah woops this got caught erroneously in my find&replace